src/Uno.UI.Tests.ViewLibrary/MyExtControl.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Tests.ViewLibrary
+{
+	public partial class MyExtControl : Control
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)